@seanyen It looks like CI ran and failed. Mind taking a look?

```
11:46:32 [  8%] Building CXX object CMakeFiles/xmlrpcpp.dir/src/XmlRpcClient.cpp.o
11:46:32 /tmp/ws/src/ros_comm/utilities/xmlrpcpp/src/XmlRpcClient.cpp: In member function ‘virtual bool XmlRpc::XmlRpcClient::generateRequest(const char*, const XmlRpc::XmlRpcValue&)’:
11:46:32 /tmp/ws/src/ros_comm/utilities/xmlrpcpp/src/XmlRpcClient.cpp:316:34: error: ‘INT_MAX’ was not declared in this scope
11:46:32   316 |   if (_request.length() > size_t(INT_MAX)) {
11:46:32       |                                  ^~~~~~~
11:46:32 /tmp/ws/src/ros_comm/utilities/xmlrpcpp/src/XmlRpcClient.cpp:14:1: note: ‘INT_MAX’ is defined in header ‘<climits>’; did you forget to ‘#include <climits>’?
11:46:32    13 | #include <string.h>
11:46:32   +++ |+#include <climits>
11:46:32    14 | 
11:46:32 /tmp/ws/src/ros_comm/utilities/xmlrpcpp/src/XmlRpcClient.cpp: In member function ‘virtual bool XmlRpc::XmlRpcClient::readHeader()’:
11:46:32 /tmp/ws/src/ros_comm/utilities/xmlrpcpp/src/XmlRpcClient.cpp:444:36: error: ‘INT_MAX’ was not declared in this scope
11:46:32   444 |   if ((clength <= 0) || (clength > INT_MAX)) {
11:46:32       |                                    ^~~~~~~
11:46:32 /tmp/ws/src/ros_comm/utilities/xmlrpcpp/src/XmlRpcClient.cpp:444:36: note: ‘INT_MAX’ is defined in header ‘<climits>’; did you forget to ‘#include <climits>’?
11:46:32 /tmp/ws/src/ros_comm/utilities/xmlrpcpp/src/XmlRpcClient.cpp: In member function ‘virtual bool XmlRpc::XmlRpcClient::readResponse()’:
11:46:32 /tmp/ws/src/ros_comm/utilities/xmlrpcpp/src/XmlRpcClient.cpp:478:37: error: ‘INT_MAX’ was not declared in this scope
11:46:32   478 |     if (_response.length() > size_t(INT_MAX)) {
11:46:32       |                                     ^~~~~~~
11:46:32 /tmp/ws/src/ros_comm/utilities/xmlrpcpp/src/XmlRpcClient.cpp:478:37: note: ‘INT_MAX’ is defined in header ‘<climits>’; did you forget to ‘#include <climits>’?
11:46:32 make[2]: *** [CMakeFiles/xmlrpcpp.dir/build.make:63: CMakeFiles/xmlrpcpp.dir/src/XmlRpcClient.cpp.o] Error 1
11:46:32 make[1]: *** [CMakeFiles/Makefile2:276: CMakeFiles/xmlrpcpp.dir/all] Error 2
```

--- a/utilities/xmlrpcpp/src/XmlRpcClient.cpp
+++ b/utilities/xmlrpcpp/src/XmlRpcClient.cpp
@@ -11,6 +11,7 @@
 	# include <strings.h>
 #endif
 #include <string.h>
+#include <climits>
 
 
 using namespace XmlRpc;

--- a/utilities/xmlrpcpp/src/XmlRpcServerConnection.cpp
+++ b/utilities/xmlrpcpp/src/XmlRpcServerConnection.cpp
@@ -11,6 +11,7 @@
 #endif
 # include <string.h>
 #endif
+#include <climits>
 
 using namespace XmlRpc;
 

--- a/utilities/xmlrpcpp/src/XmlRpcSocket.cpp
+++ b/utilities/xmlrpcpp/src/XmlRpcSocket.cpp
@@ -51,6 +51,8 @@ extern "C" {
 }
 #endif  // _WINDOWS
 
+#include <climits>
+
 #endif // MAKEDEPEND
 
 // MSG_NOSIGNAL does not exists on OS X

--- a/utilities/xmlrpcpp/src/XmlRpcUtil.cpp
+++ b/utilities/xmlrpcpp/src/XmlRpcUtil.cpp
@@ -7,6 +7,7 @@
 # include <stdarg.h>
 # include <stdio.h>
 # include <string.h>
+# include <climits>
 #endif
 
 #include "xmlrpcpp/XmlRpc.h"

--- a/utilities/xmlrpcpp/src/XmlRpcValue.cpp
+++ b/utilities/xmlrpcpp/src/XmlRpcValue.cpp
@@ -15,6 +15,7 @@
 
 #include <sstream>
 #include <mutex>
+#include <vector>
 
 namespace XmlRpc {
 

--- a/utilities/xmlrpcpp/test/TestValues.cpp
+++ b/utilities/xmlrpcpp/test/TestValues.cpp
@@ -26,6 +26,7 @@
 
 #include <stdlib.h>
 #include <string>
+#include <climits>
 
 #include "xmlrpcpp/XmlRpcValue.h"
 #include "xmlrpcpp/XmlRpcException.h"

--- a/utilities/xmlrpcpp/test/test_client.cpp
+++ b/utilities/xmlrpcpp/test/test_client.cpp
@@ -26,6 +26,7 @@
 
 #include "mock_socket.h"
 #include <errno.h>
+#include <climits>
 
 #include <gtest/gtest.h>
 

